tsdb/engine/tsm1/wal.go

```diff
@@ -285,6 +327,195 @@ func (l *WAL) newSegmentFile() error {
 	return nil
 }
 
+type WALStats struct {
+	mu sync.RWMutex
+
+	Dir string
+
+	minKey, maxKey   string
```

Comment:
I assume this is stuff that will be removed because of the changes we talked about today?